clusterloader2/testing/load/config.yaml

```diff
@@ -64,15 +64,15 @@
 {{$schedulerThroughputPodsPerDeployment := .Nodes}}
 {{$schedulerThroughputNamespaces := DivideInt $totalSchedulerThroughputPods $schedulerThroughputPodsPerDeployment}}
 
-# Set schedulerThroughputNamespaces to 1 on small clusters otherwise it will result
-# in an unnecessary number of namespaces.
-{{$schedulerThroughputNamespaces := IfThenElse $IS_SMALL_CLUSTER 1 $schedulerThroughputNamespaces}}
-
-{{if $IS_LARGE_CLUSTER}}
-  {{$schedulerThroughputReplicasPerNamespace = DivideInt .Nodes $LARGE_CLUSTER_NODES}}
+{{if $ENABLE_HUGE_SERVICES}}
```

Comment:
nope ;)

Reply:
already changed to
```
{{if and $ENABLE_HUGE_SERVICES (ge .Nodes $HUGE_SERVICES_SIZE)}}
```
the previous version was made after agreeing on it here: https://github.com/kubernetes/perf-tests/pull/1840#discussion_r658611902

Reply:
thanks

Reply:
not sure if ENABLE_HUGE_SERVICES check is needed, but I'm fine both ways